src/components/molecules/GiItem/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+const StyledImg = styled.div`
+  img {
+    border: 1em solid red;
+    border-radius: 1em;
+  }
+`;
+
+class GiItem extends Component {
+  constructor(props) {
+    super(props);
+    this.getClicked = this.getClicked.bind(this);
+  }
+
+  getClicked() {
+    this.props.toggleFavs(this.props.giphy);
+  }
+
+  render() {
+    const { giphy, selectedFavs } = this.props;
+
+    const img = giphy.images["fixed_height"];
+
+    const isFav =
+      selectedFavs !== undefined
+        ? selectedFavs.indexOf(giphy.id) !== -1
+        : false;
+
+    if (isFav) {
```

Comment:
you can create one styled component that receive a prop `isFav` and then you add border if `isFav` if not you don't do anything.
```
StyledImg = styled.img`
    ${props => props.isFav && `
            border: 1em solid red;
            border-radius: 1em;
      `}
`
```